easybuild/easyblocks/generic/conda.py

```diff
@@ -38,7 +39,7 @@
 from easybuild.tools.build_log import EasyBuildError
 
 
-class Conda(Binary):
+class CondaEnhance(Binary):
```

Comment:
```suggestion
class Conda(Binary):
```
You need to retain the original class name

Reply:
Hi ocaisa, I just fixed this class name error in the newest commit. Sorry about that.